CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 *
 
+# 1.2.0
+
+* Recognize `action`, `formaction` and `ping` as [URL attributes](https://html.spec.whatwg.org/multipage/indices.html#attributes-3) for scheme and origin filtering
```

Comment:
Why is this link inlined and not the other one ?

Reply:
No reason. 😄